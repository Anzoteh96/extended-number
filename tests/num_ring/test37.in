-750321052883602259838194489090887064837509098772131118592553488799042648880021570422371052166821549138303023363
-992902579788776562470917537660080790003643170137011253836720294341260346939991075699759263686235608250180623893
60258638066351949367986489792335350632323457893456902340215973517583870789381934179553981590300616937387284810
