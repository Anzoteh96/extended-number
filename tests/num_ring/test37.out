Enter the number base b:
Enter two numbers n and m:
b is 750321052883602259838194489090887064837509098772131118592553488799042648880021570422371052166821549138303023363
n mod b is 507739525978427957205471440521693339671375027407250983348386683256824950820052065144982840647407490026425422833
m mod b is 60258638066351949367986489792335350632323457893456902340215973517583870789381934179553981590300616937387284810
n + m is 567998164044779906573457930314028690303698485300707885688602656774408821609433999324536822237708106963812707643
n - m is 447480887912076007837484950729357989039051569513794081008170709739241080030670130965428859057106873089038138023
n * m is 182741617469727679357081956723645089422343770167840382501960317375056344808007575800228386574411871355309474851
n ** m is 411462587505824997794748878577928636009297796484053317113822595312238259988995573995033428441632836682697422381
